test/e2e/page-objects/pages/confirmations/redesign/advanced-details-transaction-confirmation.ts

```diff
@@ -0,0 +1,176 @@
+import { strict as assert } from 'assert';
+import { Driver } from '../../../../webdriver/driver';
+import { RawLocator } from '../../../common';
+import TransactionConfirmation from './transaction-confirmation';
+
+class AdvancedDetailsTransactionConfirmation extends TransactionConfirmation {
```

Comment:
These properties and methods could live directly inside `TransactionConfirmation`, since all transaction confirmations have the advanced details related to this test.

This way, one could refer to the methods on this class directly, but also from any of the confirmation screen classes that extend it, such as `SetApprovalForAllTransactionConfirmation` or `TokenTransferTransactionConfirmation`.

Reply:
done